README.md

```diff
@@ -18,23 +18,24 @@ const Persist = require('persist-store')
 // custom persisters should implement promises
 
 class LogIt {
-  save(name, value) {
+  save(value) {
     return Promise.resolve('this does nothing')
   }
-  load(name) {
-    return Promise.resolve("shh, act like i loaded something:" + name)
+  load() {
+    return Promise.resolve('shh, act like i loaded something')
   }
 }
 
 // example options
 const persist = new Persist([
     {
       type: 'local',
-      path: './data'
+      path: './data/file',
     },
     {
       type: 's3',
-      bucket: 'a_bucket',
+      bucket: 'a_bucket/some_prefix',
```

Comment:
bucket should just be bucket - s3 doesn't really have the concept of prefixes as it's just a key/value store under the hood. i.e. the prefix is part of `key`, and is really just used for listing objects with the same key prefixes. for the purpose of this, it should just be bucket/key